.github/ISSUE_TEMPLATE/report-bug.yml

```diff
@@ -7,9 +7,9 @@ assignees:
   - EdVraz
   - Astrea49
   - Toricane
+  - Catalyst4222
 body:
   - type: markdown
-    id: thanks
```

Comment:
wait a second, why was this removed?

Reply:
Pycharm said it was invalid

Reply:
did it say why?

Reply:
"key `id` not allowed here"

[some github documentation link]

Reply:
I'll check it again later

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

As you can see here markdown fields are not supposed to have an ID